### PR TITLE
DUOS-1327[risk=no]Update Researcher Console to Manage V2 endpoint

### PR DESCRIPTION
**Pull request by raejohanek**
SCOPE:
- Remove usage of /manage endpoint on Researcher Console
- Replace with /manage/v2 endpoint
- Refactor usages of data to account for the new data structure
- - remove sorting on election status (sorting on election status does not exist on the member, chair ,or admin consoles which use the same manage/v2/endpoint bc not all dars have an election)
- - update cancel button to only show when there is no election instead of being disabled for all other cases
- - update the way in which status is calculated

ADDRESSES:
partially https://broadworkbench.atlassian.net/browse/DUOS-1327

Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/pages/ResearcherConsole.js [hunk 1]
```diff
@@ -123,7 +124,7 @@ class ResearcherConsole extends Component {
 
   init() {
 
-    DAR.getDataAccessManage().then(
+    DAR.getDataAccessManageV2("Researcher").then(
```
Comment (rushtong) [inline src/pages/ResearcherConsole.js]: Can we use `USER_ROLES` in `utils.js` for the researcher value?

**Comment by rushtong** [comment]
I'm seeing this warning, but I think it's on the submitted DAR table, not the draft dar table:
```
Warning: Each child in a list should have a unique "key" prop.

Check the render method of `ResearcherConsole`. See https://reactjs.org/link/warning-keys for more information.
    at ResearcherConsole (http://localhost:3000/static/js/main.chunk.js:43474:5)
    at Route (http://localhost:3000/static/js/vendors~main.chunk.js:481701:29)
    at AuthenticatedRoute (http://localhost:3000/static/js/main.chunk.js:2333:24)
    at Switch (http://localhost:3000/static/js/vendors~main.chunk.js:481903:29)
    at Routes
    at div
    at div
    at div
    at App (http://localhost:3000/static/js/main.chunk.js:481:75)
    at Router (http://localhost:3000/static/js/vendors~main.chunk.js:481336:30)
    at BrowserRouter (http://localhost:3000/static/js/vendors~main.chunk.js:480956:35)
```

**Comment by rushtong** [comment]
Also seeing these warnings in the browser console:
```
src/pages/ResearcherConsole.js
  Line 13:34:    Missing semicolon                    @babel/semi
  Line 213:108:  There should be no space after '['   array-bracket-spacing
  Line 213:117:  There should be no space before ']'  array-bracket-spacing
```

**Review by rushtong — APPROVED** [review]
This is working for me locally, so 👍🏽 
I would like to get to the root of the keys problem ... I tried tackling it and couldn't figure it out. The best I could gather was that it was with the submitted DARs. The partial dars did not have the warning.

**Comment by raejohanek** [comment]
> This is working for me locally, so 👍🏽
> I would like to get to the root of the keys problem ... I tried tackling it and couldn't figure it out. The best I could gather was that it was with the submitted DARs. The partial dars did not have the warning.

Okay I will look into fixing it before I merge this in

**Comment by JVThomas** [comment]
It's the table header, the cells need keys. Just add a key attribute to the divs with styles in the getTableHeaderTemplateWithSort function (in DarTable)

**Pull request merged**